presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMinMaxByAggregation.java

```diff
@@ -247,7 +246,7 @@ public void testMinLongDecimalDecimal()
         assertAggregation(
                 function,
                 1.0,
-                sqlDecimal("2.2"),
+                SqlDecimal.of("2.2"),
```

Comment:
Put changes to TestMinMaxByAggregation in `"min_by", "max_by" aggregate functions for decimal(p,s)`